pkgs/applications/editors/kakoune/plugins/kak-ansi.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "kak-ansi";
+  version = "0.2.0";
+  src = fetchFromGitHub {
+    owner = "eraserhd";
+    repo = "kak-ansi";
+    rev = "v${version}";
+    sha256 = "15bw3rhwmary50j850akm766ig6klflgkily60m5zvic0rsvlk2p";
+  };
+  buildPhase = ''
+    $CC -O2 -std=c99 -o kak-ansi-filter kak-ansi-filter.c
```

Comment:
Still sceptical of this. Suppose guix wants to also have kak-ansi. They have to copy this? Why not just let people who want to precompile run make?

Reply:
ok, done